### ISSUE TITLE
Accessibility issues with p-autocomplete

### ISSUE THREAD

**Issue by dazilb** [primefaces/primeng#10506]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**

https://stackblitz.com/edit/primeng-autocomplete-demo-2e4phn

**Current behavior**

input tag does not have an aria-label for autocomplete with [multiple]="true" 
![image](https://user-images.githubusercontent.com/88963615/129474948-6c3228fd-62bf-4f8a-b7ba-8c427ce84046.png)

**Expected behavior**
add aria-labeledby or aria-label to the input tag. 

  
**Minimal reproduction of the problem with instructions**
run the stackblitz example and inspect the component.

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 5.X

* **PrimeNG version:** 11.4.4 
Also exists in latest version 

* **Browser:**all
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by yigitfindikli** [comment]
We planned full accessibility support for the next versions of PrimeNG. Thank you for your report. PrimeNG will be fully accessible soon.
